modules/cnf-configuring-nrop-on-schedlable-control-planes.adoc

```diff
@@ -0,0 +1,118 @@
+// Module included in the following assemblies:
+//
+// *scalability_and_performance/cnf-numa-aware-scheduling.adoc
+
+:_mod-docs-content-type: PROCEDURE
+
+[id="cnf-configuring-nrop-on-schedulable-cp-nodes_{context}"]
+= Configuring NUMA Resources Operator on schedulable control plane nodes
+
+
+[role="_abstract"]
+This procedure describes how to configure the NUMA Resources Operator (NROP) to manage control plane nodes that have been made schedulable. This is particularly useful in compact clusters where control plane nodes also serve as worker nodes, or in multi-node OpenShift (MNO) clusters where control plane nodes are made schedulable to run workloads.
```

Comment:
Could we just say that "are schedulable" or "are set as schedulable"? Is there a reason for saying "are made schedulable"? "are made" doesn't sound right to me and might be hard to translate so I'm wondering if there's something else that we can use here? 
```suggestion
Configure the NUMA Resources Operator (NROP) to manage control plane nodes that are schedulable. This is particularly useful in compact clusters where control plane nodes also serve as worker nodes, or in multi-node OpenShift (MNO) clusters where control plane nodes are schedulable to run workloads.
```